bot/commands/config.py

```
@@ -0,0 +1,425 @@
+            if other_options:
+                other_text = "\n".join(f"**{k}:** {v}" for k, v in other_options)
+                embed.add_field(name="Other Settings", value=other_text, inline=False)
+
+            await interaction.response.send_message(embed=embed, ephemeral=True)
+
+    def _validate_time_format(self, time_str: str) -> bool:
+        """Validate time string format (HH:MM)."""
+        try:
+            if not re.match(r'^([0-1]?[0-9]|2[0-3]):[0-5][0-9]$', time_str):
+                return False
+            return True  # Regex already validated the ranges
```

Ruff

Return the negated condition directly

(SIM103)